programmers2/두-정수사이의-합.js

```diff
@@ -0,0 +1,16 @@
+function solution(a, b) {
+    let sum = 0;
+    if (a > b) {
+        for (let i = b; i <= a; i++) 
+            sum += i;
+    }
+    else if (a < b) {
+        for (let i = a; i <= b; i++) 
+            sum += i;
+    }
+    else return a;
```

Comment:
Math.min , Math,max 를 사용하여 두 숫자의 크기를 비교하면 더욱 간결해질 수 있다.
```suggestion
    const min = Math.min(a, b);
    const max = Math.max(a, b);
    
    for (let i = min; i <= max; i++){
        result += i;
    }
```